web/packages/teleport/src/Login/MOTD/MOTD.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2023 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import { Card, Box, Text, ButtonPrimary } from 'design';
+
+export function MOTD({ message, onClick }: Props) {
+  return (
+    <Card bg="levels.surface" my={6} mx="auto" width="464px">
+      <Box p={6}>
+        <Text typography="h2" mb={3} textAlign="center" color="text.main">
+          Message of the day!
```

Comment:
Do we need a title like this? If so, should it end with an `!`?

Reply:
My thinking was to title the component and be verbose on what we are displaying below as motd message. I think that `!` is not necessary though. 

@roraback what do you think about keeping title?

Reply:
I would lean toward removing it. I do like having _a_ title, but because it isn't customizable and different customers may use this feature in quite different ways, I think it's probably best to just leave it out. In particular, I'm imagining a situation in which it's the same message for many days, and the "Message of the Day" title becomes quite ironic for users who see it day after day.

Reply:
Noted. Thanks for the input. Removed title in https://github.com/gravitational/teleport/pull/27813/commits/0995d036999bc34f211b5e399c36350ad95d39ef